src/core/trulens/core/database/sqlalchemy.py

```diff
@@ -99,6 +99,13 @@ class SQLAlchemyDB(DB):
     [ORM][trulens.core.database.orm.ORM] upon initialization.
     """
 
+    def __str__(self) -> str:
+        return f"SQLAlchemyDB({self.engine.url})"
+
+    # for DB's WithIdentString mixin
+    def _ident_str(self) -> str:
+        return self.engine.url
```

Comment:
This is not good, sometimes engine url will contain credentials?

Reply:
fixed to only show the database name